src/EditorFeatures/Core/Implementation/Classification/SemanticClassificationUtilities.cs

```diff
@@ -20,9 +28,54 @@
 
 namespace Microsoft.CodeAnalysis.Editor.Implementation.Classification
 {
-    internal static class SemanticClassificationUtilities
+    internal partial class SemanticClassifier
```

Comment:
i purposefully didn't rename the file as it causes github to lose the diff. this makes it easier to understand hte change.  i will rename in followup pr.

Reply:
At this point there's more code that's new than existed... :smile: